.github/workflows/classify.yaml

```diff
@@ -53,3 +51,16 @@ jobs:
       - uses: actions-ecosystem/action-add-labels@v1
         with:
           labels: kind/${{ steps.regex-match.outputs.group1 }}
+
+      - name: Check release notes were not deleted
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+            if (!pr.data.body.includes("```release-note")) {
+              throw new Error("Please don't cancel the ```release-note ``` section as it is used to build the release notes");
```

Comment:
nit: if the last question is valid, this is more "pr has to have a release-note section".